documentation/src/main/asciidoc/topics/con_invalidation_caches.adoc

```diff
@@ -1,11 +1,19 @@
 [id='invalidation-caches_{context}']
 = Invalidation caches
 
-You can use {brandname} in invalidation mode to optimize systems that perform high volumes of read operations.
-A good example is to use invalidation to prevent lots of database writes when state changes occur.
+Invalidation cache mode in {brandname} is designed to optimize systems that perform high volumes of read operations.
```

Comment:
Technically this is to optimize a system that has a large number of read operations to a shared external source such as a database. Invalidation cache imo makes no sense without a store.

Reply:
I updated the sentence